fsw/inc/cs_events.h

```diff
@@ -1657,106 +1657,106 @@
 /**
  * \brief CS Table Enable Checksum Unable To Find Entry In Definition Table Event ID
  *
- *  \par Type: DEBUG
+ *  \par Type: ERROR
  *
  *  \par Cause:
  *
  *  This event message is issued when CS successfully enables an entry (specified
  *  by name) in the Tables results table but is unable to find the same entry
  *  in the definition table (or the entry is marked as #CS_STATE_EMPTY).
  */
-#define CS_ENABLE_TABLE_DEF_NOT_FOUND_DBG_EID 129
+#define CS_ENABLE_TABLE_DEF_NOT_FOUND_ERR_EID 129
 
 /**
  * \brief CS Table Disable Checksum Unable To Find Entry In Definition Table Event ID
  *
- *  \par Type: DEBUG
+ *  \par Type: ERROR
  *
  *  \par Cause:
  *
  *  This event message is issued when CS successfully disables an entry (specified
  *  by name) in the Tables results table but is unable to find the same entry
  *  in the definition table (or the entry is marked as #CS_STATE_EMPTY).
  */
-#define CS_DISABLE_TABLE_DEF_NOT_FOUND_DBG_EID 130
+#define CS_DISABLE_TABLE_DEF_NOT_FOUND_ERR_EID 130
 
 /**
  * \brief CS App Enable Checksum Unable To Find Entry In Definition Table Event ID
  *
- *  \par Type: DEBUG
+ *  \par Type: ERROR
  *
  *  \par Cause:
  *
  *  This event message is issued when CS successfully enables an entry (specified
  *  by name) in the Apps results table but is unable to find the same entry
  *  in the definition table (or the entry is marked as #CS_STATE_EMPTY).
  */
-#define CS_ENABLE_APP_DEF_NOT_FOUND_DBG_EID 131
+#define CS_ENABLE_APP_DEF_NOT_FOUND_ERR_EID 131
 
 /**
  * \brief CS App Disable Checksum Unable To Find Entry In Definition Table Event ID
  *
- *  \par Type: DEBUG
+ *  \par Type: ERROR
  *
  *  \par Cause:
  *
  *  This event message is issued when CS successfully disables an entry (specified
  *  by name) in the Apps results table but is unable to find the same entry
  *  in the definition table (or the entry is marked as #CS_STATE_EMPTY).
  */
-#define CS_DISABLE_APP_DEF_NOT_FOUND_DBG_EID 132
+#define CS_DISABLE_APP_DEF_NOT_FOUND_ERR_EID 132
 
 /**
  * \brief CS Memory Disable Checksum Unable To Find Entry In Definition Table Event ID
  *
- *  \par Type: DEBUG
+ *  \par Type: ERROR
  *
  *  \par Cause:
  *
  *  This event message is issued when CS successfully disables an entry (specified
  *  by name) in the Memory results table but identifies the corresponding entry in
  *  the definitions table to be set to #CS_STATE_EMPTY.
  */
-#define CS_DISABLE_MEMORY_DEF_EMPTY_DBG_EID 133
+#define CS_DISABLE_MEMORY_DEF_EMPTY_ERR_EID 133
```

Comment:
Why was this changed to error, as opposed to info? There were a good handful of others that were also changed to error and I'm not sure why?